### DISCUSSION TITLE
Mac binary?

### DISCUSSION THREAD

**Discussion by NileshMagan** [colmap/colmap#2560] [Q&A]
Hi! Stupid question, but where and how do i access binaries for this? For each release I only see windows binaries and source code. Am i missing something obvious?

**Comment by ahojnnes** [comment]
Mac binaries are not provided anymore. You would have to follow the install instructions manually.

**Reply by orzels** [reply]
If there's not binaries for a Mac, then maybe it's time for some review of the text on main page. "Executables for Windows and Mac and other resources can be downloaded from https://demuc.de/colmap/." sounds like some stuff for Mac is on the site. But not. Weird.

**Reply by BSVogler** [reply]
Update in my PR https://github.com/colmap/colmap/pull/2622
